src/wp-includes/rest-api/endpoints/class-wp-rest-templates-controller.php

```diff
@@ -708,6 +709,14 @@ public function prepare_item_for_response( $item, $request ) { // phpcs:ignore V
 			$data['area'] = $template->area;
 		}
 
+		if ( rest_is_field_included( 'author', $fields ) ) {
+			$data['author'] = (int) $template->author;
+		}
```

Comment:
Is this proposed change for `modified` intentional, or was it committed by mistake?

Reply:
Good spotting @mukeshpanchal27 

Thank you!

I've removed it. It was just me hacking around during WIP 😅